book/security.rst

```diff
@@ -813,23 +813,25 @@ You can easily deny access from inside a controller::
 
     public function helloAction($name)
     {
-        if (false === $this->get('security.authorization_checker')->isGranted('ROLE_ADMIN')) {
-            throw $this->createAccessDeniedException();
-        }
+        // The second parameter is used to specify on what object the role is tested.
+        $this->denyAccessUnlessGranted('ROLE_ADMIN', null, 'Unable to access this page!');
+
+        // Old way :
+        // if (false === $this->isGranted('ROLE_ADMIN')) {
```

Comment:
this is not the old way given that the ``isGranted`` shortcut is also need in 2.6